public/programmerjs/receptiontransmission.js

```diff
@@ -16,52 +22,42 @@ var Paysage = Paysage || {};
     Paysage.getCompleteCodeObject(emitData);
```

Comment:
Please make sure none of the two implementations of `getCompleteCodeObject` send a `playgroundId` anymore.